packages/back-end/src/routers/safe-rollout/safe-rollout.router.ts

```diff
@@ -12,7 +15,13 @@ const snapshotParams = z.object({ id: z.string() }).strict();
 const statusBody = z
   .object({ status: z.enum(safeRolloutStatusArray) })
   .strict();
-
+const safeRolloutBody = z
+  .object({
+    safeRolloutFields: createSafeRolloutValidator,
```

Comment:
Are we sure we want the `create` validator here? Or is there a chance that we will receive just a single field to be updated for example?